virttest/qemu_vm.py

```diff
@@ -1273,7 +1273,7 @@ def add_vga(devices, vga):
             """Add primary vga device."""
             fallback = params.get("vga_use_legacy_expression") == "yes"
             machine_type = params.get("machine_type", '')
-            parent_bus = {'aobject': 'pci.0'}
+            parent_bus = self._get_pci_bus(params, 'vga')
```

Comment:
As your mentioned `Allow specify parent_bus for virtio-gpu device` from the commit message, Can I think that the priority of specifying parent_bus should be higher than the framework assign the value automatically? right?

Reply:
Yes, users should know what they are doing. All devices will be inserted into the root bus by default without users' configuration, when we define them, the priority is higher.